src/components/common/Questionnaire/ExpandedModal/NextButton.js

```diff
@@ -1,11 +1,15 @@
 import React from 'react';
 import styles from './ExpandedModal.module.css';
 import laborImage from './laborImage.png';
+import cn from 'classnames';
 
-const NextButton = ({ handleNext, buttonText }) => (
-  <div className={styles.nextStepContainer}>
+const NextButton = ({ handleNext, buttonText, isEnabled }) => (
+  <div className={styles.nextStepButtonContainer}>
     <img src={laborImage} alt="laborImage" />
-    <button onClick={handleNext} className={styles.nextStep}>
+    <button
+      onClick={handleNext}
+      className={cn(styles.nextStepButton, { [styles.enabled]: isEnabled })}
+    >
```

Comment:
Suggestion: 可以直接讓按鈕不能按
```suggestion
    <button
      onClick={handleNext}
      className={cn(styles.nextStepButton, { [styles.enabled]: isEnabled })}
      disabled={!isEnabled}
    >
```

Reply:
喔，對，直接用原生屬性做控制就好，謝謝～